### PR TITLE
fix: rewrite & follow catppuccin style guide

### PR DESCRIPTION
**Pull request by bagelwaffle**
Changes this commit makes:
use `evaluate-commands`
rename old colour names
follow current catppuccin style guide

### CODE REVIEW

**Review by NamesCode — APPROVED** [review]

File: colors/catppuccin_mocha.kak [hunk 1]
```diff
@@ -1,83 +1,82 @@
...
+        set-face global LineNumberCursor   ${rosewater},${surface2}+b
+        set-face global LineNumbersWrapped ${rosewater},${surface2}
+        set-face global MenuForeground     ${text},${surface1}+b
+        set-face global MenuBackground     ${text},${surface0}
+        set-face global MenuInfo           ${crust},${teal}
+        set-face global Information        ${crust},${teal}
+        set-face global Error              ${crust},${red}
+        set-face global StatusLine         ${text},${mantle}
+        set-face global StatusLineMode     ${crust},${yellow}
+        set-face global StatusLineInfo     ${crust}${teal}
```
Comment (chriselrod) [inline colors/catppuccin_mocha.kak]:
```suggestion
        set-face global StatusLineInfo     ${crust},${teal}
```

Comment (chriselrod) [reply colors/catppuccin_mocha.kak]: That fixes this error when using catppuccin_mocha:
```
/home/chriselrod/.config/kak/colors/catppuccin_mocha.kak:3:1: 'evaluate-commands': 29:9: 'set-face': unable to parse color: 'rgb:11111brgb:94e2d5'
Error: 1:1: 'colorscheme': 1:2: 'evaluate-commands': 1:1: 'source': 3:1: 'evaluate-commands': 29:9: 'set-face': unable to parse color: 'rgb:11111brgb:94e2d5'
```

**Comment by NamesCode** [comment]
What are the benefits of using evaluate?

**Comment by bagelwaffle** [comment]
@NamesCode

As far as I know, there are no benefits, BUT I originally wrote this with the goal of getting this merged into kakoune’s repo as well, and this was how they wrote most if not all of their themes. (See [example](https://github.com/mawww/kakoune/blob/master/colors/solarized-dark.kak))

**Comment by backwardspy** [comment]
i've just given this a quick test and it looks good. thanks for your contribution!

**Comment by backwardspy** [comment]
@bagelwaffle you now have maintainer privileges. feel free to add yourself to the "thanks to" section in the readme, and while you're there would you mind updating the preview images? thanks!

**Pull request merged**